tests/http/src/test/java/org/mule/runtime/module/http/functional/matcher/HttpMessageAttributesMatchers.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.http.functional.matcher;
+
+import org.mule.extension.http.api.HttpResponseAttributes;
+
+import org.hamcrest.Factory;
+import org.hamcrest.Matcher;
+
+public class HttpMessageAttributesMatchers
+{
+    @Factory
+    public static Matcher<HttpResponseAttributes> hasStatusCode(int statusCode)
```

Comment:
The idea is to have this constructors separate from the actual matchers so that they are all defined in one place. I expect more matchers to be created regarding headers, parts, and response/request specific attributes.